test/menagerie/roundtrip.test.ts

```
@@ -0,0 +1,137 @@
+      expect(sets[0]).toEqual(expect.arrayContaining([1, 2]));
+      expect(sets[1]).toEqual(expect.arrayContaining(['a', 'b']));
+      await expect(callValues('int_key_dict')).resolves.toEqual({ 1: 'one', 2: 'two' });
+      expect(RUNTIME_CATALOGUE.filter(row => row.status === 'KNOWN_LIE')).toHaveLength(5);
+    },
+    timeoutMs
+  );
+
+  it(
+    'fails loudly for values the transport cannot faithfully encode',
+    async () => {
+      await expect(callValues('special_floats', [true])).rejects.toThrow(/NaN|Infinity|serialize/i);
```

ESLint

[error] Move this regular expression to module scope to avoid re-compilation on every call.

(e18e/prefer-static-regex)